edge-functions/serverside-analytics/README.md

```diff
@@ -0,0 +1,52 @@
+# Serverside Analytics example
+
+This example shows how to implement serverside analytics inside [Middleware Functions](https://nextjs.org/docs/middleware).
+
+## Demo
+
+https://edge-serverside-analytics.vercel.app/
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+### One-Click Deploy
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/serverside-analytics&project-name=serverside-analytics&repository-name=serverside-analytics)
```

Comment:
We need to start adding the required env variables in the button, you can use the deploy button docs to create the link: https://vercel.com/docs/deploy-button

It needs 3 more query params, added by filling these fields:

<img width="744" alt="image" src="https://user-images.githubusercontent.com/4278345/157536820-5df80885-e51f-4add-80c8-2d69913a8975.png">

For the URL, add a link to the readme with a hash (`#`) pointing to the "Analytics Setup" title.

Reply:
Wow! Done!